src/main/java/dev/socketmods/socketnukes/client/events/ClientSNEvents.java

```diff
@@ -24,5 +28,14 @@ public static void clientSetup(FMLClientSetupEvent e) {
 
         EntityRendererManager manager = Minecraft.getInstance().getRenderManager();
         manager.register(SNRegistry.EXPLOSIVE_ENTITY_TYPE.get(), new ExplosiveEntityRenderer(manager));
+        manager.register(SNRegistry.EXPLOSIVE_BOLB_TYPE.get(), new BolbEntityRenderer(manager));
+
+        manager.getSkinMap().get("default").addLayer(new PlayerHatLayer(Minecraft.getInstance().getRenderManager().playerRenderer));
```

Comment:
get("default") returns an instance of the playerRenderer. By caching that, we can eliminate the AT used later on this line.